pkgs/development/tools/ols/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, makeWrapper, odin, lib }:
+
+stdenv.mkDerivation {
+  pname = "ols";
+  version = "20230421";
+
+  src = fetchFromGitHub {
+    owner = "DanielGavin";
+    repo = "ols";
+    rev = "fd136199897d5e5c87f6f1fbfd076ed18e41d7b7";
+    sha256 = "sha256-lRoDSc2bZSuXTam3Q5OOlSD6YAobCFKNRbtQ41Qx5EY=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  buildInputs = [
+    odin
+  ];
+
+  postPatch = ''
+    patchShebangs build.sh
+  '';
+
+  buildPhase = ''
+    ./build.sh
```

```suggestion
    odin build src/ -collection:shared=src -out:ols -o:speed
```
this sounds like it is going to do native optimisation. We must turn this off.

---

it won't - you'd need to pass the `-microarch:native` flag to odin for that to happen